states/FL/governor.yaml

```diff
@@ -1,6 +1,6 @@
 contact_form:
   steps:
-    - visit: "https://www.flgov.com/contact-gov-scott/email-the-governor/"
+    - visit: "https://www.flgov.com/contact-gov-desantis/email-the-governor/"
```

Comment:
Might as well make this the even more generic https://www.flgov.com/email-the-governor/.

Reply:
Done